src/test/java/DeadlineTest.java

```diff
@@ -0,0 +1,39 @@
+import Duke.task.Deadline;
+import Duke.task.Task;
+import org.junit.jupiter.api.Test;
+
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DeadlineTest {
+
+    /**
+     * Testing method
+     */
+    @Test
+    public void LocalDateFormatTest() {
```

Comment:
```suggestion
    public void localDateFormat_validDate_success() {
```
I think you can use the three part format for test method names?